docs/api/session.md

```diff
@@ -504,6 +512,7 @@ The `uploadData` is an array of `data` objects:
 * `data` Object
   * `bytes` Buffer - Content being sent.
   * `file` String - Path of file being uploaded.
+  * `blobUUID` String - UUID of blob data.
```

Comment:
It can probably be a `blob` object that has a method to get length and read the data.